src/components/widgets/DoItYourself.vue

```diff
@@ -105,6 +113,8 @@ import { useAppInterfaceStore } from '@/stores/appInterface'
 import { useWidgetManagerStore } from '@/stores/widgetManager'
 import type { Widget } from '@/types/widgets'
 
+const autoSave = ref(false)
```

Comment:
Maybe we could use this with a `useBlueOsStorage` composable and it would be saved in the localStorage for that user, so they don't need to toggle the checkbox every time they open the DIY widget window:

```typescript
const autoSave = useBlueOsStorage('diy-widget-auto-save', false)
```